src/transformers/models/paligemma/processing_paligemma.py

```diff
@@ -70,6 +71,11 @@ def build_string_from_input(prompt, bos_token, image_seq_len, image_token):
         image_seq_len (`int`): The length of the image sequence.
         image_token (`str`): The image token.
     """
+    if image_token in prompt:
+        warnings.warn(
+            f"The image token {image_token} is already present in the prompt. This may lead to unexpected behavior."
```

Comment:
We should mention explicitly here that the next line will replace image_token by an empty string